### ISSUE TITLE
skipping questions

### ISSUE THREAD

**Issue by JPPinon** [wecitizens/regional-gps-app#25]
La longueur totale du questionnaire pour un utilisateur dépend de sa Région, mais fera environ 55 questions. Il serait bon qu'il puisse interrompre l'interrogatoire et allez aux résultats. (En réalité, en cliquant sur résultats, l'utilisateur tombe sur la page "stats" de même que s'il avait cliquer sur "stats". Idéalement, cette possibilité devrait être activée à partir de la 20 question seulement. Cela peut se faire en affichant un message (en bas d'écran):
"En cliquant ci-dessus sur le bouton "résultats", vous pouvez interrompre l'interrogatoire. Sachez toutefois que la précision des résultats sera bien meilleure, si vous répondez à toutes les questions."
"Door hierboven op de knop "resultaten" te klikken, kunt u de bevraging stoppen. Houd er echter rekening mee dat de nauwkeurigheid van de resultaten veel beter is als u alle vragen beantwoordt."
"By clicking above on the "results" button, you can interrupt the questioning. Be aware, however, that the accuracy of the results will be much better if you answer all the questions.

**Comment by danielsum** [comment]
@JPPinon => la personne peut skipper les questions (ici pour le test on a mis dès la 2ème question mais ça sera à la 20ème) ? 

=> je rajoute juste une notice qq part en bas avec le texte donné au dessus de la navigation "précédent - Aller aux résultats - suivants" => c'est ça ?

**Comment by JPPinon** [comment]
Oui. En fait, mon texte ci-dessus pourrait remplacer la mention existante "Allez aux résultats" en bas de l'écran, à condition que le texte "4. Résultats" soit actif.

**Comment by danielsum** [comment]
Done.

**Comment by JPPinon** [comment]
OK. Mais l'idée serait qu'en cliquant sur "Résultats" l'utilisateur soit dirigé vers la page de stats. L'effet serait donc le même qu'ile clique sur "stats" out "résultats".

**Comment by danielsum** [comment]
C’est le cas actuellement?

Ça mène vers stats

--
Daniel Sum

**Comment by JPPinon** [comment]
Aujourd'hui, en cliquant sur "Résultats" j'aboutis sur le résultats sans passer par les stats.

En remplaçant la mention "voir les résultats" par la phrase demandée, tu as supprimé les flèches à gauche et à droite (pour passer à la page précédente ou suivante). C'est mieux laisser les flèches.

**Comment by danielsum** [comment]
Les flèches sont laissés voir : https://www.evernote.com/l/AHcBtyVUlPBLFrBJfaC17BXvaHYcBE_TsJY

Par contre effectivement on sait cliquer en haut sur Results (j'avais pas du tout compris ça) => j'essaye de corriger ça ce soir.

Dan

**Comment by JPPinon** [comment]
Je ne voyait pas les flèches parce qu'elles se trouvaient en-dehors de mon écran (laptop !).
Ne pourrait-on pas supprimer l'énorme espace blanc entre les bouton de vote et la suite.

**Comment by JPPinon** [comment]
Je résume ce qui m'intéresserait d'urgence: Quand on clique sur résultats (dans la barre d'avancement du haut), il faut envoyer l'utilisateur dans la page de stats.  MUST
Vérifier que les stats sont bien enregistrés.
Réduire l'espace mort entre les boutons de vote et les flèches (page up / page down). SHOULD
Chez moi, on ne voit pas les flèches:
![image](https://user-images.githubusercontent.com/43548775/58281036-5f62df80-7da2-11e9-96a0-a2dc8fac7a01.png)

**Comment by JPPinon** [comment]
@danielsum  Merci Dan: le bouton "résultats" va maintenant vers la page de stats. Tu regardes encore les 2 autres choses mentionnées ci-dessus?
